_targets.R

```diff
@@ -20,23 +47,15 @@ p1_targets_list <- list(
 
 p2_targets_list <- list(
   tar_target(
-    site_data_clean, 
-    process_data(site_data)
-  ),
-  tar_target(
-    site_data_annotated,
-    annotate_data(site_data_clean, site_filename = site_info_csv)
-  ),
-  tar_target(
-    site_data_styled,
-    style_data(site_data_annotated)
+    site_data_munged, 
+    process_data(site_data, site_info_csv)
   )
 )
 
 p3_targets_list <- list(
   tar_target(
     figure_1_png,
-    plot_nwis_timeseries(fileout = "3_visualize/out/figure_1.png", site_data_styled),
+    plot_nwis_timeseries(fileout = "3_visualize/out/figure_1_temp_ts.png", site_data_munged),
```

Comment:
Since you have renamed the file being saved here, I would suggest updating the name of this target, too. So `figure_1_temp_ts_png` (see my previous comment about naming conventions for file targets).